api/config/config_local.json

```diff
@@ -99,7 +99,12 @@
     "unsafeAllowUserCreationFromGSuiteData": true,
     "enableRasLoginGovLinking": true,
     "enforceRasLoginGovLinking": true,
-    "currentDuccVersions": [3, 4]
+    "currentDuccVersions": [3, 4],
```

Comment:
Outside of scope of this PR: Why can't we have a common config json that contains common configs? Then extended by env config file? It's a good code practice to avoid duplication.

Reply:
That's a great idea.  I don't know.  Let's talk about this at Eng Sync.